Gemfile.lock

```diff
@@ -1,60 +1,60 @@
 GEM
```

Comment:
There are a very large number of gems in this file. Have you had a chance to look at them to determine which ones you need?

Reply:
After investigation and a reinstall, I couldn't find any that are not needed